package.json

```diff
@@ -65,9 +65,13 @@
     "prettier:templates": "npx prettier --write './build/components/render/*.html'"
   },
   "lint-staged": {
-    "./src/**/*.{js,scss}": [
+    "./src/**/*.js": [
       "npx prettier --write",
       "npx eslint",
+      "git add"
+    ],
+    "./src/**/*.scss": [
+      "npx prettier --write",
       "npx stylelint",
       "git add"
```

Comment:
The `git add` is no longer required as of `lint-staged@v10+`.

>v10
From v10.0.0 onwards any new modifications to originally staged files will be automatically added to the commit. If your task previously contained a git add step, please remove this. The automatic behaviour ensures there are less race-conditions, since trying to run multiple git operations at the same time usually results in an error.

**source** https://www.npmjs.com/package/lint-staged#v10

Reply:
Fixed in 2e31968.